### ISSUE TITLE
MEL integration BeginScope + Serilog doesn't include data as Sentry Tag

### ISSUE THREAD

**Issue by bruno-garcia** [getsentry/sentry-dotnet#340]
In the Controller of: https://github.com/getsentry/symbol-collector
Only using `IHub.ConfigureScope` I was able to get data into Sentry tags. Using `logger.BeginScope` the data was not included.

**Comment by Xocix** [comment]
I have the same issue with MEL + Sentry, without Serilog. Anyone has made this work with `logger.beginScope`?

**Comment by zeqk** [comment]
I have the same problem with MEL. 
net471
Sentry.AspNetCore Version="2.1.5" 
Microsoft.AspNetCore Version="2.2.0"
(no Serilog)

**Comment by zeqk** [comment]
I found the solution for my project
```csharp
var builder = WebHost.CreateDefaultBuilder(args)
                //...
                .ConfigureLogging((ctx, logging) =>
                {
                    logging.AddSentry(); //Use this
                })
                //..
               //.UseSentry() //Don't use this
                .UseStartup<Startup>();
```

and if you are using the `appsettings.json` for configure Sentry, you need to use the Logging configuration

```json
{
  "Logging": {
      "Sentry": {
          "IncludeScopes": true,
          "Dsn": "https://url...",
           ...
        "LogLevel": {
             ...
         }
      }
   }
}
```

**Comment by bruno-garcia** [comment]
This should have been fixed in version 2.1.8 https://github.com/getsentry/sentry-dotnet/releases/tag/2.1.8
